osu.Game.Tests/Visual/Navigation/TestSceneStartupBeatmapDisplay.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Linq;
+using NUnit.Framework;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Testing;
+using osu.Game.Overlays;
+
+namespace osu.Game.Tests.Visual.Navigation
+{
+    public class TestSceneStartupBeatmapDisplay : OsuGameTestScene
```

Comment:
I find it kinda weird that this test scene kind of just... doesn't work, either in debug or release config - sure, the overlay loads, but it's blank, because the `TestOsuGame` is using the provided dummy `API` so requests will never actually succeed.

Thoughts on mocking the requests to make this test actually show something?